yaml_edit/test/remove_test.dart

```diff
@@ -48,6 +48,20 @@ c: 3
     });
   });
 
+  group('returns', () {
+    test('returns the removed node when successful', () {
+      final doc = YamlEditor('{ a: { b: foo } }');
+      final node = doc.remove(['a', 'b']);
+      expect(node.value, equals('foo'));
+    });
+
+    test('returns null-value node when doc is empty and path is empty', () {
+      final doc = YamlEditor('');
+      final node = doc.remove([]);
+      expect(node.value, equals(null));
+    });
+  });
```

Comment:
Tests to cover `remove()` return values.

Reply:
Feel free to break additional tests into a separate PR.. those are easy to land independently :D